systems/controllers/controlUtil.cpp

```diff
@@ -283,8 +284,8 @@ MatrixXd individualSupportCOPs(RigidBodyManipulator* r, const std::vector<Suppor
 
       Vector3d point_on_contact_plane = contact_positions.col(0);
       std::pair<Vector3d, double> cop_and_normal_torque = resolveCenterOfPressure(torque, force, normal, point_on_contact_plane);
-      Vector4d cop_body;
-      cop_body << cop_and_normal_torque.first, 1.0;
+      Vector3d cop_body;
+      cop_body << cop_and_normal_torque.first;
       Vector3d cop_world;
       r->forwardKin(active_support.body_idx, cop_body, 0, cop_world);

```

Comment:
Could even just use `cop_and_normal_torque.first` instead of `cop_body` in this call and get rid of `cop_body`